languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -22406,6 +22406,34 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- ====================================================================== -->
     <category id="COLLOQUIALISMS" name="Umgangssprache" type="register">
         <!-- TODO Make one Java-rule for context-independent, colloquial expressions and maintain a list -->
+        <rulegroup id="ZUM_STATT_FUERS" name="fürs (zum) Schneiden">
+            <rule id="ZUM_STATT_FUERS1" name="fürs (zum) Schneiden1">
+                <pattern>
+                    <marker>
+                        <token>fürs<exception regexp='yes' scope='previous'>danke|danke schön</exception></token>
+                    </marker>
+                    <token postag='SUB.*INF.*' postag_regexp='yes'></token>
+                    <token skip="-1" min="0" inflected='yes' regexp='yes'>brauchen|benötigen|nehmen</token>
+                </pattern>
+                <message>Hier könnte <suggestion>zum</suggestion> besser klingen, da fürs sehr umgangssprachlich ist.</message>
```

Comment:
Metasprache bitte immer durch einfache Anführungszeichen markieren (in dem Fall 'fürs')